src/errors.ts

```diff
@@ -18,3 +18,20 @@ export function getNotExistErrorResponse(attribute: string, path: string): Promi
             variables: { path }
         }));
 }
+
+export function randomPortWithoutUserDataDir() {
+    return Promise.reject(new ErrorWithMessage(<DebugProtocol.Message>{
+        id: 2008,
+        format: localize('random.port.no.userdatadir', 'When the remote debugging port is set to 0, you must also provide the "userDataDir" launch argument'),
+        sendTelemetry: true
+    }));
+}
+
+export function couldNotConnectToPort(address: string, port: number) {
+    return Promise.reject(new ErrorWithMessage(<DebugProtocol.Message>{
+        id: 2008,
+        format: localize('launch.port.not.open', 'Could not open a connection to Chrome DevTools at: {address}:{port}', '{address}', '{port}'),
```

Comment:
Would it be clearer for the user if we say we are connecting to just "Chrome"?

Reply:
done